src/components/layout/TransactionWorkflow/TransactionWorkflow.tsx

```diff
@@ -235,6 +235,13 @@ function TransactionWorkflow({
                 3: { title: 'Complete', status: '' },
               };
             }
+            case PDNT_INTERACTION_TYPES.SET_TTL_SECONDS: {
+              return {
+                1: { title: 'Confirm TTL Seconds', status: 'pending' },
+                2: { title: 'Deploy TTL Seconds Change', status: '' },
+                3: { title: 'Complete', status: '' },
+              };
+            }
             default:
```

Comment:
still my opinon we put this switch case into an object in a `constants` file

Reply:
moved this the transaction data object and created an accessor util